content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -822,12 +824,23 @@ involved.
 
 ### Synchronous and asynchronous instruments
```

Comment:
@svrnm @cartermp et al.: I think that the title of this section should use the term "instrumentation" not "instruments". WDYT?

I'm asking here because your answer will impact the text being added/changed here.

Reply:
@austinlparker ☝️

Reply:
It should remain as "instruments". Confusingly, it has a specific meaning in the context of metrics that's distinct from other signals.

Reply:
Ah, ... just read the spec and the definition of [instrument](https://opentelemetry.io/docs/reference/specification/metrics/api/#instrument) -- so now I better understand the context. I'm still wondering if it would be better to use the term "instrumentation" in the title here.

Reply:
Actually, nm, I'm dropping this thread (marking as resolved). Sorry for the noise.